test/test_gem2rpm.rb

```diff
@@ -36,6 +36,19 @@ def test_show_message_for_obj
     assert_equal("foo\n\nbar\n", @out.string)
   end
 
+  def test_show_templates
+    Gem2Rpm.show_templates(@out)
+    out_str = @out.string
```

Comment:
I don't think it will provide any measurable difference. Please drop the temporary variable.